libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -74,6 +72,12 @@ class AC_AttitudeControl {
     // Empty destructor to suppress compiler warning
     virtual ~AC_AttitudeControl() {}
 
+    // set_dt / get_dt - dt is the time since the last time the attitude controllers were updated
+    // _dt should be set based on the time of the last IMU read used by these controllers
+    // the attitude controller should run updates for active controllers on each loop to ensure normal operation
+    void set_dt(float dt) { _dt = dt; }
+    float get_dt() const { return _dt; }
```

Comment:
I don't immediately see any places where we call AC_AttitudeControl::get_dt().  I do see calls to AC_PosControl::get_dt().

It's again not a blocker but we don't normally add methods that aren't used.

Reply:
Yes, there was a place I needed to call this instead of the Position controller.